gabc/index.html

```diff
@@ -374,23 +344,27 @@ <h2 id="text">Text</h2>
+<p>In Gregorian chant, aligning the neumes with the text is important so that the singer can match them up appropriately.  Gregorio automates this using the rules described on <a href="../graphy.html" title="">the page about the writing of chant</a>. However, this may not give the desireable result in all cases.  If it is necessary, put the part of the text to be aligned between braces.  For instance, in the following example the neumes will align with the center of &#8220;ve&#8221; instead of the first &#8220;e&#8221;:</p>
+
+<div class="commandline">
+<code>e{ve}r(e)</code>
+</div>
 
 <h3 id="initial">Initial determination</h3>
 
-<p>The initial is automatically recognized by gregorio as the first letter. If this behaviour is not the one you want, you can force the text centering of the first syllable. Thus Gregorio will consider that the letters preceding the centered text are the initial (it can be empty), and the centered letter as well as the following ones will be under the first note. You can also use the gabc attribute <code>initial-style:0;</code> in the header.</p>
+<p>The initial is automatically recognized by gregorio as the first letter.  You can use the gabc attribute <code>initial-style</code> in the header to set its size.  The values corespond to number of lines the intial takes up: <code>0</code> (there is not initial), <code>1</code> (the default), or <code>2</code> (the initial takes up two lines).</p>
 
-<p>Note that in the case of a syllable composed of one letter only, gregorio does not automatically add an hyphen under the first notes. This is due to the fact that it's easy to add it in gabc, and it would be difficult to remove it if it was automatically added.</p>
+<p>Note that in the case of a syllable composed of one letter only, Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> automatically adds a hyphen under the first notes.  To disable this feature, use the command <code>\gresetemptyfirstsyllablehyphen{auto}</code>.</p>
 
 <h3 id="specialcharacters">Special characters</h3>
 
-<p>It can happen that the texts of Gregorian chants contain non-unicode characters, so we must consider how to notate them.   Special characters used in Gregorian notation are <code>ý</code>, <code>ǽ</code>, <code>œ́</code>, the crossed R (for <em>Response</em>), the crossed V (for <em>Versicle</em>) and the crossed A (for <em>Antiphon</em>). Of these characters, only the crossed A is not to be found in the Unicode table.</p>
+<p>It can happen that the texts of Gregorian chants contain unicode characters which are not easily accesible from the keyboard.   Examples of this sort of characters used in Gregorian notation are <code>ý</code>, <code>ǽ</code>, <code>œ́</code>, the crossed R (℟ for <em>Response</em>), and the crossed V (℣ for <em>Versicle</em>).  There is also the possibility of a non-unicode character such as the crossed A (for <em>Antiphon</em>).</p>
```

Comment:
`this sort of characters` should probably be `this sort of character`.  `used in Gregorian notation` is redundant because of the previous sentence and (I think) should be omitted.  Also, we call these "barred" letters elsewhere (and in the TeX code itself), so I think `crossed` here should be changed to `barred`.